com.ibm.streamsx.topology/opt/python/include/splpy_tuple.h

```diff
@@ -36,14 +48,56 @@ namespace streamsx {
 
   /**
    * Convert the SPL tuple that represents a Python object
-   * to a Python tuple that holds:
-   *  - pickled value as a memory view object
+   * to a Python tuple that holds as its first argument.
+   * One or two arguments in a Python tuple are passed to the function
+   * (which is a wrapper around the user function).
```

Comment:
There are some grammatical errors and missing words which make this section difficult to understand:
* `to a Python tuple that holds as its first argument.` should end in a colon
* `One or two arguments in a Python tuple are passed to the function` I think should be ` `One or two arguments in a Python tuple which are passed to the function`